packages/governance/src/question.js

```diff
@@ -80,6 +83,14 @@ const looksLikeParamChangeIssue = issue => {
   assertInstance(issue.contract);
 };
 
+const looksLikeApiInvocation = issue => {
```

Comment:
The type annotations in the switch aren't doing what I suspect they were meant to. I think you're after  [Assertion Functions](https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#assertion-functions) .

```suggestion
/**
 * @param {*} issue
 * @returns {asserts issue is ApiInvocationIssue}
 */
const looksLikeApiInvocation = issue => {
```

Reply:
done